### PR TITLE
Adding in notice about default use of SRI

### PR DESCRIPTION
**Pull request by jonathanKingston**
Is part of:
https://github.com/ember-cli/rfcs/pull/20

### CODE REVIEW

**Comment by stefanpenner** [comment]
Will need to be merged when we release next

**Comment by joostdevries** [comment]
I think this needs to be trimmed down to just mentioning it's included and delegating everything else to the addon docs.

**Comment by stefanpenner** [comment]
i would keep the why + what, but defer the how to the addons site.

**Comment by joostdevries** [comment]
:+1:

**Comment by jonathanKingston** [comment]
Heh that means I should patch the README of the current addon to the latest fixes here (yup out of sync is totally an issue here). Give me 20 minutes and I will publish a 1.0.2 with the latest here and cut this down.

It should not impact the blueprint change as it references: `^1.0.1`

**Comment by stefanpenner** [comment]
@jonathanKingston sorry about that

**Comment by jonathanKingston** [comment]
@stefanpenner don't be I was thinking it was on the long side myself anyway, thanks.

1.0.2 published and updated the changes here: https://github.com/jonathanKingston/ember-cli/commit/b45716165f45f37934b9a14f34fb2c5a0851fe6d

Is that better @joostdevries and @stefanpenner ?

**Comment by stefanpenner** [comment]
I feel good about it, @joostdevries has a better eye for these things, but I suspect this address his concerns.

**Comment by joostdevries** [comment]
@jonathanKingston Looks better. My main concern is with first-time ember users getting the feeling they have to learn learn a dozen new concepts before they can start building.

File: _posts/2013-04-09-asset-compilation.md [hunk 1]
```diff
@@ -346,3 +346,23 @@ var app = new EmberApp({
...
+#### Why
+The reason to add this to your application is to protect against poisoned CDNs breaking JavaScript or CSS.
+
+- [JavaScript DDoS prevention](https://blog.cloudflare.com/an-introduction-to-javascript-based-ddos/)
+  - The latest [GitHub DDoS attack](http://googleonlinesecurity.blogspot.co.uk/2015/04/a-javascript-based-ddos-attack-as-seen.html)
+- Protection against corrupted code on less trusted servers
+
+#### Configure
+
+To configure SRI generation see: [ember-cli-sri](https://github.com/jonathanKingston/ember-cli-sri)
```
Comment (joostdevries) [inline _posts/2013-04-09-asset-compilation.md]: I'd change this to `to customize SRI generation`.

**Comment by jonathanKingston** [comment]
@joostdevries yup the intent is to actually let users stumble over it.

Updated with the latest 'customize' text change.

**Comment by joostdevries** [comment]
:+1:

**Comment by stefanpenner** [comment]
Ah. This should have waited for the next release

**Comment by joostdevries** [comment]
@stefanpenner whoops, sorry about this. Can also revert?

**Comment by stefanpenner** [comment]
> @stefanpenner whoops, sorry about this. Can also revert?

not a big deal, more for future syncing (another thing that will be tricky once we add multiple release channels)

**Pull request merged**